Automattic/jetpack
# VideoPress block: Extract poster-related functionality from useSyncMedia hook

Refactor part of preparation for implementing a native version of the `useSyncMedia` hook. Note, this is the second of a series of planned PRs for splitting the hook's functionality into smaller hooks, with the goal of making it more straightforward to implement the native functionality. The first of these PRs can be found at https://github.com/Automattic/jetpack/pull/29898.

See https://github.com/wordpress-mobile/gutenberg-mobile/issues/5611 for more context.

## Proposed changes:

* https://github.com/Automattic/jetpack/pull/29973/commits/8c1804bca5c93c405a0c04905fdfaba3fe906639: The functionality related to fetching the video frame poster in the `useSyncMedia` hook has been extracted. For this purpose, a hook named `useVideoPosterData` has been created.

### Other information:


## Jetpack product discussion

p9ugOq-3mV-p2

## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

Ensure it's possible to walk through the testing steps outlined in https://github.com/Automattic/jetpack/pull/29746, and that there are no regressions.
